src/api/handlers/auth.py

```diff
@@ -42,10 +42,9 @@ def login(self, request):
     # if the signin is from an admin site then set it to 24 hrs
     if(context.is_admin_site):
       MAX_AGE = 86400
-    
-    response.set_cookie("token", value=token, max_age=MAX_AGE)    
-    return response
 
+    response.set_cookie("token", value=token, max_age=MAX_AGE, samesite='Lax')    
```

Comment:
why Lax?

Reply:
@Opoku-Agyemang I wasn't sure what to put.  Should it be "Strict" or "None"?  It was complaining in the console.